src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -123,6 +123,59 @@ def updateAgentDrainingMode(self, agentName, drainFlag):
                                    (drainFlag, resp))
             return False
 
+    def getCampaignConfig(self, campaignName):
+        """
+        get campaign config for transferor function in unified ReqMgr2MS.
+        """
+        return self._getDataFromMemoryCache('campaignconfig/%d' % campaignName)
+
+    def postCampaignConfig(self, campaignName, campaignConfig):
+        """
+
+        :param campaignName
+        :type basestringg
+        :param campaignConfig:
+        :type dict - only can replace whole campaign  no partial parameters
+
+        campaignName: "HIRun2015":
+        campaignConfig: {
+            "go": true,
+            "labels" : ["02May2016","25Aug2016"],
+            "overflow" : {"PRIM" : {}},
+            "DDMcopies": {
+                         "all" : { "N" : 2 }
+            },
+            "custodial_override" : ["DQMIO"],
+            "fractionpass": 1.0,
+            "lumisize" : -1,
+            "maxcopies" : 1,
+            "custodial": "T1_FR_CCIN2P3_MSS",
+            "parameters" :{
+              "NonCustodialSites" : ["T2_US_Vanderbilt"],
+              "SiteBlacklist": [
+              "T1_US_FNAL",
+              "T2_US_Purdue",
+              "T2_US_Caltech",
+              "T2_US_Florida",
+              "T2_US_Nebraska",
+              "T2_US_UCSD",
+              "T2_US_Wisconsin"
+             ]
+           }
+          }
+
+        :return: None
+        """
+        return self["requests"].post('campaignconfig' % campaignName, campaignConfig)[0]['result']
```

Comment:
Aren't you missing the document name here? Something like
` return self["requests"].post('campaignconfig/%s' % campaignName, campaignConfig)[0]['result'] `

Reply:
You are right, updated now.